docs/BigQueryCreateDataset-action.md

```diff
@@ -0,0 +1,56 @@
+# BigQuery Create Dataset
+
+#### **Description**
+
+Creates a BigQuery Dataset.
+
+#### **Usage**
+
+Dataset names must be unique per project. The dataset name contain up to 1,024 characters and can
+contain letters (upper or lower case), numbers and underscore. Dataset names cannot contain special characters
+such as '-', '&', '*', or '%'.
+
+If dataset name starts with '_' (underscore) then the dataset would be hidden and not visible in
+the view and dataset names are case sensitive (E.g. 'dataset' and 'Dataset' are two different
+datasets that can co-exists in a project.
+
```

Comment:
As mentioned in the original PR, there should be a description about what happens if the dataset already exists. Do the description and labels get updated? Do they get ignored for whatever currently exists?